src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiVariableConfigurationEditor.js

```diff
@@ -0,0 +1,214 @@
+import {FieldArray, useFormikContext} from 'formik';
+import isEqual from 'lodash/isEqual';
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useAsync, useToggle} from 'react-use';
+
+import {APIContext} from 'components/admin/form_design/Context';
+import {REGISTRATION_OBJECTS_TARGET_PATHS} from 'components/admin/form_design/constants';
+import Field from 'components/admin/forms/Field';
+import Fieldset from 'components/admin/forms/Fieldset';
+import FormRow from 'components/admin/forms/FormRow';
+import {TextInput} from 'components/admin/forms/Inputs';
+import Select, {LOADING_OPTION} from 'components/admin/forms/Select';
+import {post} from 'utils/fetch';
+
+import {asJsonSchema} from './utils';
+
+/**
+ * Returns the Objects API Configuration editor modal for a specific variable. This only applies to V2 Options
+ *
+ * @typedef {{
+ *   version: 1 | 2;
+ *   objecttype: string;
+ *   objecttypeVersion: number;
+ *   variablesMapping: {variableKey: string, targetPath: string[]}[];
+ * }} ObjectsAPIRegistrationBackendOptions
+ *
+ * @param {Object} p
+ * @param {Object} p.variable - The current variable
+ * @returns {JSX.Element} - The summary, represented as a the parts of the target path separated by '>'
+ */
+const ObjectsApiVariableConfigurationEditor = ({variable}) => {
+  const {csrftoken} = useContext(APIContext);
+
+  const [jsonSchemaVisible, toggleJsonSchemaVisible] = useToggle(false);
+  const {values: backendOptions, getFieldProps, setFieldValue} = useFormikContext();
```

Comment:
```suggestion
  const {values: backendOptions, getFieldProps} = useFormikContext();
```
I don't think it's used here?